staking/integration-tests/src/governance/instructions.rs

```diff
@@ -0,0 +1,191 @@
+use {
+    crate::{
+        solana::utils::fetch_account_data,
+        staking::{
+            instructions::get_update_voter_weight_instruction,
+            pda::{
+                get_config_address,
+                get_max_voter_record_address,
+                get_voter_record_address,
+            },
+        },
+    },
+    litesvm::types::TransactionResult,
+    solana_sdk::{
+        pubkey::Pubkey,
+        signature::Keypair,
+        signer::Signer,
+        transaction::Transaction,
+    },
+    spl_governance::state::{
+        proposal::{
+            get_proposal_address,
+            VoteType,
+        },
+        token_owner_record::get_token_owner_record_address,
+        vote_record::{
+            Vote,
+            VoteChoice,
+        },
+    },
+    staking::state::{
+        global_config::GlobalConfig,
+        voter_weight_record::VoterWeightAction,
+    },
+};
+
+pub fn create_token_owner_record(svm: &mut litesvm::LiteSVM, payer: &Keypair) -> TransactionResult {
+    let config = get_config_address();
+
+    let config_data: GlobalConfig = fetch_account_data(svm, &config);
+    let pyth_governance_realm = &config_data.pyth_governance_realm;
+    let governance_program = &config_data.governance_program;
+    let pyth_token_mint = &config_data.pyth_token_mint;
+
+    let ix = spl_governance::instruction::create_token_owner_record(
+        governance_program,
+        pyth_governance_realm,
+        &payer.pubkey(),
+        pyth_token_mint,
+        &payer.pubkey(),
+    );
+
+    let tx = Transaction::new_signed_with_payer(
+        &[ix],
+        Some(&payer.pubkey()),
+        &[&payer],
+        svm.latest_blockhash(),
+    );
+
+    svm.send_transaction(tx)
+}
+
+pub fn create_proposal(
```

Comment:
I had this convention in mind that each function in `instructions.rs` corresponds to exactly 1 instruction. And in `helper_functions.rs` we can call multiple of these instructions in a single function.

Reply:
I separated sign-off from create proposal, but I am going to keep get_update_voter_weight inside create_proposal since they're always supposed to be called together